sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -770,12 +770,10 @@ final void moveToOutputFiles(
       }
       // During a failure case, files may have been deleted in an earlier step. Thus
       // we ignore missing files here.
-      FileSystems.rename(
-          srcFiles,
-          dstFiles,
-          StandardMoveOptions.IGNORE_MISSING_FILES,
```

Comment:
We should make sure that not passing these options do not end up being a regression for other file systems (S3, HDFS, local, etc.)

Reply:
Moved back, as get operations have different quota than writes the filtering might be beneficial.